### DISCUSSION TITLE
Is it possible to @unique arrays?

### DISCUSSION THREAD

**Discussion by nullndr** [prisma/prisma#14027] [Q&A] [answered]
Let's say I have the following model:

```prisma
model User {
  id                 String @id @default(cuid())
  whitelistedDomains String[]
}
```

Can I make the `whitelistedDomains` a unique array (`whitelistedDomains String[] @unique`)?

If yes, what's the behaviour with the empty array (`[]`)?

**Comment by austincrim** [comment, marked as answer]
Hey @nullndr 👋,

It is possible to use `@unique` with scalar lists.

An error will be thrown if you try to save multiple rows with an empty list `[]` (I tested with Postgres, it might vary by database provider).

It seems we don't have this documented well at the moment, I will pass that on to our docs team 👍.

**Reply by nullndr** [reply]
Thank you so much, hope y'all the best

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
